utilsplot.py

```diff
@@ -573,6 +573,21 @@ def rga_plot(G, w_start=-2, w_end=2, axlim=None, points=1000, fig=0,
     freqresp = [G(si) for si in s]
 
     plot_No = 1
+    
+    plotting_order = {
+        'outputs' : {
```

Comment:
Spacing is still strange. Should be `'value': key` (no space before colon, one space after) everywhere.

Reply:
Now PEP8 compliant.